### ISSUE TITLE
Insufficient color contrast for the post list title badge and the post card title badge

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#66322]
### Description

Some special Pages (and potentially other objects edited in the editor) show a 'badge' with an additional label to clarify the special nature fo the edited object. For example: The page set as home page and the page set as posts page.

These badges belong to different components but they use the same design pattern. The color contrast is insufficient gor both:
Text color: `#757575`
Background color: `#f0f0f0`

That gives a [color contrast of only 4.04:1](https://jdlsn.com/color/?type=hex&color=757575&color2=f0f0f0). It should be at least 4.6:1

To help search through the c odebase, the CSS selectors for these are:
`edit-site-post-list__title-badge`
`editor-post-card-panel__title-badge`

Cc @WordPress/gutenberg-design 

### Step-by-step reproduction instructions

- Go to the wp-admin Reading Settings.
- Set your homepage to display a static page.
- Set a Posts page.
- Go to the Site editor > Pages.
- In the list of pages, search for the pages used for the homepage and posts page.
- Observe they both show a 'badge' with an additional label.
- Observe the color contrast of the badge is insufficient.
- Select one of these special pages.
- In the Settings panel, observe the color contrast of the badge in the post card panel is insufficient.

### Screenshots, screen recording, code snippet

![Image](https://github.com/user-attachments/assets/bd376344-dcca-47bc-9fe0-475a86038171)

![Image](https://github.com/user-attachments/assets/ab6f1ac1-0313-4f16-9d03-522733ec335c)

![Image](https://github.com/user-attachments/assets/8a11570d-ec09-431f-bec6-e909740ab065)

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by Vrishabhsk** [comment]
Hi @afercia 👋 
- I have raised a `PR` to address this issue : https://github.com/WordPress/gutenberg/pull/66360
- The `PR` increased the `color-contrast` from [`4.04:1`](https://www.joedolson.com/tools/color-contrast.php?type=hex&color=757575&color2=f0f0f0) to [`11.75:1`](https://www.joedolson.com/tools/color-contrast.php?type=hex&color=2f2f2f&color2=f0f0f0)
- I'm not sure if a big jump like this `desirable`, since I went with the defined variables for `colors` in [`_colors.scss`](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.scss#L10)

Let me know your thoughts on this. Thanks

**Comment by afercia** [comment]
Tbanks @Vrishabhsk I will ask the design team on the PR.
